projects/objects/lights/protos/ConstructionLamp.proto

```diff
@@ -94,10 +94,10 @@ PROTO ConstructionLamp [
       }
```

Comment:
The SpotLight need to be converted (View ->Optional Rendering -> Show light position can help)